consensus/src/consensus_observer/observer/active_state.rs

```diff
@@ -42,6 +42,9 @@ pub struct ActiveObserverState {
     // The current epoch state
     epoch_state: Option<Arc<EpochState>>,
 
+    // Execution pool window size (if none, execution pool is disabled)
+    execution_pool_window_size: Option<u64>,
```

Comment:
Hmm we're using `u64` for the `window_size` in `OnChainConsensusConfig`. But once it's deserialized and in memory, we're treating it as a `usize` (or likely will be changed to `Option<usize>`). (See the `window_size` field in [BlockStore in Brian's commit here](https://github.com/aptos-labs/aptos-core/blob/aad1f510fef11d9221125286fbe3d2548c419ac5/consensus/src/block_storage/block_store.rs#L84-L102))

I'm not sure if this is going to be a problem but just wanted to point it out. For the sake of consistency I think probably should be the same which means maybe we change the type in Brian's PR cc @bchocho

Reply:
SGTM 😄 I'll change this one to `Option<usize>`.

Reply:
oops, I had a pending comment here.
I think `Option<u64>` actually makes more sense. I guess @hariria and me have to come to some agreement :)